src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/DrawingView.shared.cs

```diff
@@ -94,5 +97,19 @@ static object CoerceValue(BindableObject bindable, object value)
 			Color strokeColor,
 			Color backgroundColor) =>
 			DrawingViewService.GetImageStream(points.ToList(), imageSize, lineWidth, strokeColor, backgroundColor);
+
+		public void OnDrawingCompleted()
+		{
+			if (!Points.Any())
+				return;
+
+			if (DrawingCompletedCommand?.CanExecute(null) ?? false)
```

Comment:
CanExecute(Points)

Reply:
I thought it would get fixed in a separate PR for this: #1474 but I guess it is a limited change. I thought additional unit tests would have been good though which then increases the size of that change.

Reply:
you are adding a new code, where actually the bug exists. even if it's just a copy from another part. I think it would be nice to fix it as well

Reply:
OK Happy to fix. I will add some unit tests covering the bug fix also

Reply:
This should now be addressed